functions/Import-DbaCsv.ps1

```diff
@@ -443,7 +443,7 @@ function Import-DbaCsv {
 
                     # boundary case: ensure the correct db is referenced in the $server object
                     if ($Database -ne $server.ConnectionContext.DatabaseName) {
```

Comment:
@lancasteradam In order to keep the logic and error messages in sync, shouldn't we be checking the `CurrentDatabase` property instead of the `DatabaseName` property? Having a `if -ne` check on one property then throwing an error reporting the value of another property feels like it's going to lead to headaches for users.

Reply:
@sandcap-sc, great catch, I'll make that change.